adding-module.rst

```diff
@@ -159,58 +166,77 @@ modules already included in MicroPython. I will not be covering this in detail.
 Classes
 =======
 
-Let's try to add a class to our module. A class is similar to a module -- it's
-also a C struct with certain fields:
+A class is a C-struct with certain fields, quite similar to a module:
```

Comment:
This is a change in the style of the text. I thought this would fit the rest of the style of the text better.